src/test/java/chapter5/Question1Test.java

```diff
@@ -0,0 +1,19 @@
+package chapter5;
+
+import org.junit.jupiter.api.Test;
+import src.chapter5.question1.Question1;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class Question1Test {
+    @Test
+    public void getInternedString_whenAStringIsInterned_shouldReturnTrue() {
+        assertTrue(Question1.isInternedString("hello"));
+    }
+
+    @Test
+    public void getInternedStringBuilder_whenAStringIsNotInterned_shouldReturnFalse() {
```

Comment:
The first part of the test name is incorrect.

Reply:
The test name has been modified.